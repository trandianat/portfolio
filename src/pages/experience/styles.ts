import { css } from '@emotion/react';

export const experience = css({
  display: 'flex',
  flexDirection: 'column',
  gap: 24,
  '.sans': { fontSize: '.875rem', letterSpacing: 1 },
});

export const group = {
  border: '1px dotted black',
  padding: '16px 24px',
  '.company': { fontFamily: 'var(--serif)' },
};

export const groupJobs = css({
  display: 'flex',
  flexDirection: 'column',
  gap: 24,
  marginTop: 22,
});
