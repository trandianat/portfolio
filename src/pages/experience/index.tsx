import { Content } from 'components/content';
import { Job } from 'components/job';
import * as styles from 'pages/experience/styles';

export const Experience = (): JSX.Element => (
  <Content>
    <div css={styles.experience}>
      <Job
        role="Software Development Engineer III"
        company="Haven Technologies"
        start="JUL '21"
        end="AUG '23"
        skills={[
          'React',
          'Redux',
          'TypeScript',
          'Angular',
          'CSS',
          'Emotion',
          'Storybook',
          'Handlebars',
          'Jest',
          'Cypress',
          'React Testing Library',
          'Node.js',
          'Express',
          'PostgreSQL',
          'Knex.js',
          'Docker',
          'Kubernetes',
          'Agile',
          'SaaS',
        ]}
        scope="Served as a frontend subject matter expert (SME) for annuity, disability insurance, and claims product teams. Fostered professional growth and performed annual reviews for 5 full-stack developers across 2 teams."
        accomplishments={[
          'Led architectural design and development of accessibility, authentication, and notifications for claims applications, reducing the average claim process from 1 month manually to 1 week digitally for the primary client, MassMutual',
          'Promoted to Chapter Lead (software engineering manager) in 2022',
        ]}
        indent
      />
      <div css={styles.group}>
        <h2 className="company">Liberty Mutual</h2>
        <p className="sans">
          Technical Development Program & Post-Program Placement
        </p>
        <div css={styles.groupJobs}>
          <Job
            role="Senior Software Engineer"
            company="Direct Sales & Claims"
            start="OCT '17"
            end="JUL '21"
            skills={[
              'React',
              'Redux',
              'TypeScript',
              'Node.js',
              'Express',
              'GraphQL',
              'DynamoDB',
              'MongoDB',
              'Jest',
              'Enzyme',
              'Lambdas',
              'Agile',
            ]}
            scope="Iterated upon quote and claims applications with fail-fast mentality by rapidly launching A/B tests. Assisted in career growth as a mentor for junior software engineers and interns."
            accomplishments={[
              'Improved conversion rate by ideating on and leading development of a quote status feature and a guided policy builder across 4 micro-frontends',
              'Expanded customer base beyond first-party claimants by scaling claims application to accommodate third-party claimants and carriers',
              'Boosted customer confidence by creating an application to guide different types of claimants in deciding whether to file a claim',
            ]}
          />
          <Job
            role="Software Developer"
            company="Sales Compensation & Reinsurance"
            start="FEB '16"
            end="SEP '17"
            skills={['AngularJS', 'Java', 'Gradle', 'Oracle', 'MySQL', 'Agile']}
            scope="Monitored and ensured frictionless monthly insurance agent compensation transactions and statements. Enhanced insurer productivity by developing reinsurance claims, underwriting, and contact management applications."
            accomplishments={[
              'Reduced company spending and optimized processing speed by migrating sales compensation batch jobs from on-premise servers to the cloud',
              'Researched and implemented unit testing solution for reinsurance applications',
            ]}
          />
          <Job
            role="Technology Associate"
            company="Publishing & RPA (Reporting, Planning & Analytics)"
            start="JUN '14"
            end="FEB '16"
            skills={['Dojo', 'Java', 'SOAP', 'XML', 'SharePoint', 'Waterfall']}
            scope="Selected for rotational entry-level Technical Development Program. Designated as Technical Lead on a team with 4 fellow program participants."
            accomplishments={[
              'Promoted policy-issuing efficiency by integrating a formerly manual process of gathering form templates into a centralized document repository portal',
              'Increased company-wide engagement in organizational lean transformation efforts by participating in change initiatives and publishing knowledge-sharing articles',
            ]}
          />
        </div>
      </div>
      <Job
        role="Operations Intern"
        company="Community Economic Development Assistance Corporation"
        start="JUN '13"
        end="MAY '14"
        skills={['HTML', 'CSS', 'Adobe Photoshop']}
        scope="Managed office operations by maintaining contact management system, and answering incoming phone calls and directing them to the appropriate extensions."
        accomplishments={[
          'Raised community awareness of corporate mission by crafting and distributing engaging emails and event invitations with on-brand graphics',
        ]}
        indent
      />
    </div>
  </Content>
);
