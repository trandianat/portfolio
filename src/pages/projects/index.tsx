import { Link, Outlet, useLocation } from 'react-router-dom';
import Book from 'assets/icons/book';
import Chat from 'assets/icons/chat';
import Letter from 'assets/icons/letter';
import Note from 'assets/icons/note';
import Photo from 'assets/icons/photo';
import Wedding from 'assets/icons/wedding';
import Auth from 'components/auth';
import { Content } from 'components/content';
import * as styles from 'pages/projects/styles';
import isProjects from 'utils/projects';

type Project = {
  description: string;
  icon: JSX.Element;
  name: string;
  path: string;
};

export const Projects = (): JSX.Element => {
  const { pathname } = useLocation();
  const projects: Project[] = [
    {
      description:
        'Create, read, update, and delete notes sorted by latest update',
      icon: <Note />,
      name: 'Notes',
      path: 'notes',
    },
    {
      description: 'Detect objects in images by percentages of confidence',
      icon: <Photo />,
      name: 'Detection',
      path: 'detection',
    },
    {
      description:
        'Conversational artificial intelligence for scheduling appointments',
      icon: <Chat />,
      name: 'Chatbot',
      path: 'chatbot',
    },
    {
      description:
        'A website for my wedding guests to RSVP and find more information',
      icon: <Wedding />,
      name: 'Wedding',
      path: 'wedding',
    },
    {
      description:
        "Search for New York Times' best-selling books by date and category",
      icon: <Book />,
      name: 'Books',
      path: 'books',
    },
    {
      description: 'A recreation of the popular New York Times word game',
      icon: <Letter />,
      name: 'Wordle',
      path: 'wordle',
    },
  ];
  return (
    <Content>
      <div css={styles.auth}>
        <Auth>
          {isProjects(pathname) && (
            <div css={styles.projects}>
              {projects.map(({ description, icon, name, path }: Project) => (
                <Link css={styles.card} key={name} to={path}>
                  {icon}
                  <div css={styles.project}>
                    <h2>{name}</h2>
                    <p className="description">{description}</p>
                  </div>
                </Link>
              ))}
            </div>
          )}
          <Outlet />
        </Auth>
      </div>
    </Content>
  );
};
