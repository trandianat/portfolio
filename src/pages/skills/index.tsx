import { Fragment } from 'react';
import { Background } from 'components/background';
import { Header } from 'components/header';
import { Navigation } from 'components/navigation';
import * as styles from 'pages/skills/styles';
import { Color } from 'utils/types';

export const Skills = (): JSX.Element => {
  const primary = [
    'React',
    'JavaScript',
    'TypeScript',
    'HTML',
    'CSS/Sass',
    'PostgreSQL',
    'AWS',
    'REST APIs',
    'Git',
    'CI/CD',
    'Agile',
  ];
  const familiar = [
    'Accessibility',
    'Node.js',
    'MongoDB',
    'Webpack',
    'Docker',
    'Kubernetes',
    'Redux',
    'Angular',
    'Java',
    'GraphQL',
  ];
  return (
    <Fragment>
      <Header />
      <Background color={Color.GRAY}>
        <Navigation />
        <div css={styles.skills}>
          <div>
            <h2>Primarily used:</h2>
            <ul>
              {primary.map((skill: string) => (
                <li key={skill}>{skill}</li>
              ))}
            </ul>
          </div>
          <div>
            <h2>Familiar with:</h2>
            <ul>
              {familiar.map((skill: string) => (
                <li key={skill}>{skill}</li>
              ))}
            </ul>
          </div>
        </div>
      </Background>
    </Fragment>
  );
};
