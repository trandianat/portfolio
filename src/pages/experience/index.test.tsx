import { ReactNode } from 'react';
import { useLocation } from 'react-router-dom';
import { create } from 'react-test-renderer';
import { Experience } from 'pages/experience';

jest.mock('react-router-dom', () => ({
  Link: ({ children, to }: { children: ReactNode; to: string }) => (
    <a href={to}>{children}</a>
  ),
  useLocation: jest.fn(),
}));

describe('Experience', () => {
  const mockUseLocation = useLocation as jest.Mock;
  mockUseLocation.mockReturnValue({ pathname: '/' });

  it('should match the snapshot', () => {
    expect(create(<Experience />)).toMatchSnapshot();
  });
});
