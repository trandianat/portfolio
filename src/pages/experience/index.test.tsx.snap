// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience should match the snapshot 1`] = `
[
  .emotion-0 {
  background-color: var(--white);
  padding: 48px 0;
  width: 100%;
}

.emotion-1 {
  margin: 0 auto;
  width: var(--desktop);
}

@media (max-width: 950px) {
  .emotion-1 {
    width: 650px;
  }
}

@media (max-width: 700px) {
  .emotion-1 {
    width: 400px;
  }
}

@media (max-width: 450px) {
  .emotion-1 {
    width: 80%;
  }
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 36px;
}

.emotion-3 {
  background-color: var(--black);
  height: 2px;
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
}

@media (max-width: 700px) {
  .emotion-4 {
    -webkit-flex-direction: column-reverse;
    -ms-flex-direction: column-reverse;
    flex-direction: column-reverse;
    gap: 24px;
  }
}

@media (max-width: 360px) {
  .emotion-4 h1 {
    line-height: 72px;
    margin-bottom: 12px;
  }
}

.emotion-5 {
  background-color: var(--black);
  border-radius: 50%;
  color: var(--white);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 2rem;
  height: 100px;
  letter-spacing: 3px;
  min-height: 100px;
  min-width: 100px;
  width: 100px;
}

.emotion-5 .content {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 6px;
  margin: auto;
}

.emotion-5 .dot {
  background-color: var(--white);
  border-radius: 50%;
  height: 4px;
  margin: 0 auto;
  width: 4px;
}

.emotion-5 .line {
  background-color: var(--white);
  height: 1px;
  margin: 0 auto;
  width: 24px;
}

@media (max-width: 700px) {
  .emotion-5 {
    margin: 0 auto;
  }
}

@media (min-width: 951px) {
  .emotion-6 {
    display: none;
  }
}

.emotion-7 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  gap: 4px;
  vertical-align: bottom;
}

<div
    className="emotion-0"
  >
    <div
      className="emotion-1"
    >
      <div
        className="emotion-2"
      >
        <div
          className="emotion-3"
        />
        <div
          className="emotion-4"
        >
          <div>
            <h1>
              Diana Tran
            </h1>
            <p
              className="sans"
            >
              Software Engineer
            </p>
          </div>
          <div
            className="emotion-5"
          >
            <div
              className="content"
            >
              <div
                className="dot"
              />
              <div>
                DT
              </div>
              <div
                className="line"
              />
            </div>
          </div>
        </div>
        <p>
          Detail-oriented documentation expert, eager to fail fast, learn, and iterate. Experienced in design-thinking ideation and building seamless, accessible, and confidence-boosting user experiences. Well versed in the software development lifecycle including architecture and system design, testing, and monitoring.
           
          <span
            className="emotion-6"
          >
            <a
              className="emotion-7"
              href="https://www.linkedin.com/in/trandianat"
              target="_blank"
            >
              LinkedIn
              <svg
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 3c-1.093 0-2 .907-2 2v14c0 1.093.907 2 2 2h14c1.093 0 2-.907 2-2v-7h-2v7H5V5h7V3H5zm9 0v2h3.586l-9.293 9.293 1.414 1.414L19 6.414V10h2V3h-7z"
                />
              </svg>
            </a>
             
            &
             
            <a
              className="emotion-7"
              href="https://github.com/trandianat"
              target="_blank"
            >
              Github
              <svg
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 3c-1.093 0-2 .907-2 2v14c0 1.093.907 2 2 2h14c1.093 0 2-.907 2-2v-7h-2v7H5V5h7V3H5zm9 0v2h3.586l-9.293 9.293 1.414 1.414L19 6.414V10h2V3h-7z"
                />
              </svg>
            </a>
          </span>
        </p>
      </div>
    </div>
  </div>,
  .emotion-0 {
  background-color: var(--gray);
  padding: 48px 0;
  width: 100%;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.emotion-1 {
  margin: 0 auto;
  width: var(--desktop);
}

@media (max-width: 950px) {
  .emotion-1 {
    width: 650px;
  }
}

@media (max-width: 700px) {
  .emotion-1 {
    width: 400px;
  }
}

@media (max-width: 450px) {
  .emotion-1 {
    width: 80%;
  }
}

.emotion-2 {
  background-color: var(--black);
  border: 1px solid var(--black);
  box-shadow: var(--shadow);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: -48px 0 36px;
  position: -webkit-sticky;
  position: sticky;
  text-align: center;
  top: 0;
  z-index: 1;
}

.emotion-2 a {
  color: var(--white);
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  font-weight: normal;
  padding: 12px 0;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-2 a svg {
  fill: var(--white);
}

.emotion-2 a.active {
  background-color: var(--white);
  color: var(--black);
  font-weight: 500;
  pointer-events: none;
}

.emotion-2 a:hover {
  background-color: var(--white);
  color: var(--black);
}

.emotion-2 a:hover svg {
  fill: var(--black);
}

@media (max-width: 950px) {
  .emotion-2 .external {
    display: none;
  }
}

@media (max-width: 700px) {
  .emotion-2 a {
    font-size: .875rem;
  }
}

@media (max-width: 450px) {
  .emotion-2 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }

  .emotion-2 a {
    letter-spacing: 4px;
    padding: 10px 0;
  }

  .emotion-2 a:nth-of-type(-n + 2) {
    border-bottom: 1px solid var(--white);
  }
}

.emotion-3 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  gap: 6px;
  vertical-align: bottom;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 24px;
}

.emotion-5 .sans {
  font-size: .875rem;
  letter-spacing: 1px;
}

.emotion-6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 16px;
}

@media (min-width: 451px) {
  .emotion-6 {
    padding-left: 24px;
  }
}

@media (max-width: 450px) {
  .emotion-6 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    gap: 2px;
  }
}

.emotion-7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 12px;
}

@media (max-width: 450px) {
  .emotion-7 {
    display: none;
  }
}

.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  font-weight: 500;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  min-width: 70px;
  padding: 8px 0;
  text-align: right;
  white-space: nowrap;
}

.emotion-9 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 8px 0;
}

.emotion-10 {
  background-color: var(--white);
  border: 2px solid var(--dark-gray);
  border-radius: 8px;
  min-height: 8px;
  min-width: 8px;
}

.emotion-11 {
  background-color: var(--dark-gray);
  height: 100%;
  width: 2px;
}

.emotion-13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  font-weight: 500;
  gap: 2px;
}

@media (min-width: 451px) {
  .emotion-13 {
    display: none;
  }
}

.emotion-14 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
}

.emotion-16 {
  background-color: var(--dark-gray);
  height: 2px;
  width: 100%;
}

.emotion-18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
}

.emotion-19 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-20 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  gap: 6px;
  margin: 6px 0;
}

.emotion-21 {
  background-color: var(--black);
  border-radius: 12px;
  color: var(--white);
  font: .685rem var(--sans);
  letter-spacing: 1px;
  padding: 2px 8px;
  text-transform: uppercase;
}

.emotion-41 {
  border: 1px dotted black;
  padding: 16px 24px;
}

.emotion-41 .company {
  font-family: var(--serif);
}

.emotion-42 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 24px;
  margin-top: 22px;
}

.emotion-43 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 16px;
}

@media (max-width: 450px) {
  .emotion-43 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    gap: 2px;
  }
}

<div
    className="emotion-0"
  >
    <div
      className="emotion-1"
    >
      <nav
        className="sans emotion-2"
      >
        <a
          href="/"
        >
          Experience
        </a>
        <a
          href="/skills"
        >
          Skills
        </a>
        <a
          href="/projects"
        >
          Projects
        </a>
        <a
          className="external emotion-3"
          href="https://www.linkedin.com/in/trandianat"
          target="_blank"
        >
          <svg
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M5 3c-1.093 0-2 .907-2 2v14c0 1.093.907 2 2 2h14c1.093 0 2-.907 2-2v-7h-2v7H5V5h7V3H5zm9 0v2h3.586l-9.293 9.293 1.414 1.414L19 6.414V10h2V3h-7z"
            />
          </svg>
          LinkedIn
        </a>
        <a
          className="external emotion-3"
          href="https://github.com/trandianat"
          target="_blank"
        >
          <svg
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M5 3c-1.093 0-2 .907-2 2v14c0 1.093.907 2 2 2h14c1.093 0 2-.907 2-2v-7h-2v7H5V5h7V3H5zm9 0v2h3.586l-9.293 9.293 1.414 1.414L19 6.414V10h2V3h-7z"
            />
          </svg>
          Github
        </a>
      </nav>
      <div
        className="emotion-5"
      >
        <div
          className="emotion-6"
        >
          <div
            className="emotion-7"
          >
            <div
              className="emotion-8"
            >
              <div
                className="sans"
              >
                AUG '23
              </div>
              <div
                className="sans"
              >
                JUL '21
              </div>
            </div>
            <div
              className="emotion-9"
            >
              <div
                className="emotion-10"
              />
              <div
                className="emotion-11"
              />
              <div
                className="emotion-10"
              />
            </div>
          </div>
          <div
            className="emotion-13"
          >
            <div
              className="emotion-14"
            >
              <div
                className="emotion-10"
              />
              <div
                className="emotion-16"
              />
              <div
                className="emotion-10"
              />
            </div>
            <div
              className="emotion-18"
            >
              <div
                className="sans"
              >
                JUL '21
              </div>
              <div
                className="sans"
              >
                AUG '23
              </div>
            </div>
          </div>
          <div
            className="emotion-19"
          >
            <h2>
              Software Development Engineer III
            </h2>
            <p
              className="sans"
            >
              Haven Technologies
            </p>
            <div
              className="emotion-20"
            >
              <div
                className="emotion-21"
              >
                React
              </div>
              <div
                className="emotion-21"
              >
                Redux
              </div>
              <div
                className="emotion-21"
              >
                TypeScript
              </div>
              <div
                className="emotion-21"
              >
                Angular
              </div>
              <div
                className="emotion-21"
              >
                HTML
              </div>
              <div
                className="emotion-21"
              >
                CSS
              </div>
              <div
                className="emotion-21"
              >
                Emotion
              </div>
              <div
                className="emotion-21"
              >
                Storybook
              </div>
              <div
                className="emotion-21"
              >
                Handlebars
              </div>
              <div
                className="emotion-21"
              >
                Jest
              </div>
              <div
                className="emotion-21"
              >
                Cypress
              </div>
              <div
                className="emotion-21"
              >
                React Testing Library
              </div>
              <div
                className="emotion-21"
              >
                Node.js
              </div>
              <div
                className="emotion-21"
              >
                Express
              </div>
              <div
                className="emotion-21"
              >
                PostgreSQL
              </div>
              <div
                className="emotion-21"
              >
                Knex.js
              </div>
              <div
                className="emotion-21"
              >
                Docker
              </div>
              <div
                className="emotion-21"
              >
                Kubernetes
              </div>
              <div
                className="emotion-21"
              >
                Agile
              </div>
              <div
                className="emotion-21"
              >
                SaaS
              </div>
            </div>
            <p>
              Served as a frontend subject matter expert (SME) for annuity, disability insurance, and claims product teams. Fostered professional growth and performed annual reviews for 5 full-stack developers across 2 teams.
            </p>
            <ul>
              <li>
                Led architectural design and development of accessibility, authentication, and notifications for claims applications, reducing average claim process from 1 month manually to 1 week digitally for primary client, MassMutual.
              </li>
              <li>
                Promoted to Chapter Lead (software engineering manager) in 2022.
              </li>
            </ul>
          </div>
        </div>
        <div
          className="emotion-41"
        >
          <h2
            className="company"
          >
            Liberty Mutual
          </h2>
          <p
            className="sans"
          >
            Technical Development Program & Post-Program Placement
          </p>
          <div
            className="emotion-42"
          >
            <div
              className="emotion-43"
            >
              <div
                className="emotion-7"
              >
                <div
                  className="emotion-8"
                >
                  <div
                    className="sans"
                  >
                    JUL '21
                  </div>
                  <div
                    className="sans"
                  >
                    OCT '17
                  </div>
                </div>
                <div
                  className="emotion-9"
                >
                  <div
                    className="emotion-10"
                  />
                  <div
                    className="emotion-11"
                  />
                  <div
                    className="emotion-10"
                  />
                </div>
              </div>
              <div
                className="emotion-13"
              >
                <div
                  className="emotion-14"
                >
                  <div
                    className="emotion-10"
                  />
                  <div
                    className="emotion-16"
                  />
                  <div
                    className="emotion-10"
                  />
                </div>
                <div
                  className="emotion-18"
                >
                  <div
                    className="sans"
                  >
                    OCT '17
                  </div>
                  <div
                    className="sans"
                  >
                    JUL '21
                  </div>
                </div>
              </div>
              <div
                className="emotion-19"
              >
                <h2>
                  Senior Software Engineer
                </h2>
                <p
                  className="sans"
                >
                  Direct Sales & Claims
                </p>
                <div
                  className="emotion-20"
                >
                  <div
                    className="emotion-21"
                  >
                    React
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Redux
                  </div>
                  <div
                    className="emotion-21"
                  >
                    TypeScript
                  </div>
                  <div
                    className="emotion-21"
                  >
                    HTML
                  </div>
                  <div
                    className="emotion-21"
                  >
                    CSS
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Node.js
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Express
                  </div>
                  <div
                    className="emotion-21"
                  >
                    GraphQL
                  </div>
                  <div
                    className="emotion-21"
                  >
                    DynamoDB
                  </div>
                  <div
                    className="emotion-21"
                  >
                    MongoDB
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Jest
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Enzyme
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Lambdas
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Agile
                  </div>
                </div>
                <p>
                  Iterated upon quote and claims applications with fail-fast mentality by rapidly launching A/B tests. Assisted in career growth as mentor for junior software engineers and interns.
                </p>
                <ul>
                  <li>
                    Improved conversion rate by ideating on and leading development of quote status feature and guided policy builder across 4 micro-frontends.
                  </li>
                  <li>
                    Expanded customer base beyond first-party claimants by scaling claims application to accommodate third-party claimants and carriers.
                  </li>
                  <li>
                    Created application to guide different types of claimants in deciding whether to file claim, boosting customer confidence.
                  </li>
                </ul>
              </div>
            </div>
            <div
              className="emotion-43"
            >
              <div
                className="emotion-7"
              >
                <div
                  className="emotion-8"
                >
                  <div
                    className="sans"
                  >
                    SEP '17
                  </div>
                  <div
                    className="sans"
                  >
                    FEB '16
                  </div>
                </div>
                <div
                  className="emotion-9"
                >
                  <div
                    className="emotion-10"
                  />
                  <div
                    className="emotion-11"
                  />
                  <div
                    className="emotion-10"
                  />
                </div>
              </div>
              <div
                className="emotion-13"
              >
                <div
                  className="emotion-14"
                >
                  <div
                    className="emotion-10"
                  />
                  <div
                    className="emotion-16"
                  />
                  <div
                    className="emotion-10"
                  />
                </div>
                <div
                  className="emotion-18"
                >
                  <div
                    className="sans"
                  >
                    FEB '16
                  </div>
                  <div
                    className="sans"
                  >
                    SEP '17
                  </div>
                </div>
              </div>
              <div
                className="emotion-19"
              >
                <h2>
                  Software Engineer
                </h2>
                <p
                  className="sans"
                >
                  Sales Compensation & Reinsurance
                </p>
                <div
                  className="emotion-20"
                >
                  <div
                    className="emotion-21"
                  >
                    AngularJS
                  </div>
                  <div
                    className="emotion-21"
                  >
                    HTML
                  </div>
                  <div
                    className="emotion-21"
                  >
                    CSS
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Java
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Gradle
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Oracle
                  </div>
                  <div
                    className="emotion-21"
                  >
                    MySQL
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Agile
                  </div>
                </div>
                <p>
                  Monitored and ensured frictionless monthly insurance agent compensation transactions and statements. Enhanced insurer productivity by developing reinsurance claims, underwriting, and contact management applications.
                </p>
                <ul>
                  <li>
                    Reduced company spending and optimized processing speed by migrating sales compensation batch jobs from on-premise servers to cloud.
                  </li>
                </ul>
              </div>
            </div>
            <div
              className="emotion-43"
            >
              <div
                className="emotion-7"
              >
                <div
                  className="emotion-8"
                >
                  <div
                    className="sans"
                  >
                    FEB '16
                  </div>
                  <div
                    className="sans"
                  >
                    JUN '14
                  </div>
                </div>
                <div
                  className="emotion-9"
                >
                  <div
                    className="emotion-10"
                  />
                  <div
                    className="emotion-11"
                  />
                  <div
                    className="emotion-10"
                  />
                </div>
              </div>
              <div
                className="emotion-13"
              >
                <div
                  className="emotion-14"
                >
                  <div
                    className="emotion-10"
                  />
                  <div
                    className="emotion-16"
                  />
                  <div
                    className="emotion-10"
                  />
                </div>
                <div
                  className="emotion-18"
                >
                  <div
                    className="sans"
                  >
                    JUN '14
                  </div>
                  <div
                    className="sans"
                  >
                    FEB '16
                  </div>
                </div>
              </div>
              <div
                className="emotion-19"
              >
                <h2>
                  Technology Associate
                </h2>
                <p
                  className="sans"
                >
                  Publishing & RPA (Reporting, Planning & Analytics)
                </p>
                <div
                  className="emotion-20"
                >
                  <div
                    className="emotion-21"
                  >
                    HTML
                  </div>
                  <div
                    className="emotion-21"
                  >
                    CSS
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Dojo
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Java
                  </div>
                  <div
                    className="emotion-21"
                  >
                    SOAP
                  </div>
                  <div
                    className="emotion-21"
                  >
                    XML
                  </div>
                  <div
                    className="emotion-21"
                  >
                    SharePoint
                  </div>
                  <div
                    className="emotion-21"
                  >
                    Waterfall
                  </div>
                </div>
                <p>
                  Selected for rotational entry-level Technical Development Program. Designated as Technical Lead on a team with 4 fellow program participants.
                </p>
                <ul>
                  <li>
                    Promoted policy-issuing efficiency by integrating formerly manual process of gathering form templates into centralized document repository portal.
                  </li>
                  <li>
                    Participated in change initiatives and published knowledge-sharing articles, increasing company-wide engagement in organizational lean transformation efforts.
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
        <div
          className="emotion-6"
        >
          <div
            className="emotion-7"
          >
            <div
              className="emotion-8"
            >
              <div
                className="sans"
              >
                MAY '14
              </div>
              <div
                className="sans"
              >
                JUN '13
              </div>
            </div>
            <div
              className="emotion-9"
            >
              <div
                className="emotion-10"
              />
              <div
                className="emotion-11"
              />
              <div
                className="emotion-10"
              />
            </div>
          </div>
          <div
            className="emotion-13"
          >
            <div
              className="emotion-14"
            >
              <div
                className="emotion-10"
              />
              <div
                className="emotion-16"
              />
              <div
                className="emotion-10"
              />
            </div>
            <div
              className="emotion-18"
            >
              <div
                className="sans"
              >
                JUN '13
              </div>
              <div
                className="sans"
              >
                MAY '14
              </div>
            </div>
          </div>
          <div
            className="emotion-19"
          >
            <h2>
              Operations Intern
            </h2>
            <p
              className="sans"
            >
              Community Economic Development Assistance Corporation
            </p>
            <div
              className="emotion-20"
            >
              <div
                className="emotion-21"
              >
                HTML
              </div>
              <div
                className="emotion-21"
              >
                CSS
              </div>
              <div
                className="emotion-21"
              >
                Adobe Photoshop
              </div>
            </div>
            <p>
              Managed office operations, maintained contact management system, and answered incoming phone calls, directing to appropriate extensions.
            </p>
            <ul>
              <li>
                Raised community awareness of corporate mission by crafting and distributing engaging emails and event invitations with on-brand graphics.
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
